spec/models/ability_spec.rb

```diff
@@ -143,29 +145,28 @@
         end
       end
 
+      describe "team's students or admin should be able to mark attendance to a conference" do
+        context 'when user is a student from a team and try to update attendance' do
 
-      describe "a user should not be able to mark another's attendance to a conference" do
-
-        context 'when same user' do
-          let!(:attendance) { FactoryGirl.create(:attendance, user: user)}
+          let!(:user) { FactoryGirl.create(:student) }
+          let!(:attendance) { FactoryGirl.create(:attendance, team_id: user.student_team.id) }
 
           it 'allows marking of attendance' do
-            expect(ability).to be_able_to(:crud, attendance)
+            expect(ability).to be_able_to(:crud, user)
           end
 
-
           context 'when user is admin' do
             let!(:organiser_role) { FactoryGirl.create(:organizer_role, user: user)}
             it "should be able to crud attendance" do
-              expect(subject).to be_able_to :crud, attendance
+              expect(subject).to be_able_to(:crud, organiser_role)
```

Comment:
Again, I don't understand what this is testing 🙈 The description reads different to the expectation...